sql/range_column_expr.go

```diff
@@ -320,6 +346,12 @@ func (r RangeColumnExpr) DebugString() string {
 // intersection result is not the empty RangeColumnExpr, however a valid RangeColumnExpr is always returned if the error
 // is nil.
 func (r RangeColumnExpr) TryIntersect(other RangeColumnExpr) (RangeColumnExpr, bool, error) {
+	if other.Type() == RangeType_Null || r.Type() == RangeType_Null {
```

Comment:
We're explicitly handling `All` in the index builder, but not here?

Reply:
yeah i can cut this i think